r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -711,3 +711,23 @@ test_that("am/pm mirror lubridate", {
   )
 
 })
+
+test_that("date works in arrow", {
+  # https://issues.apache.org/jira/browse/ARROW-13168
+  skip_on_os("windows")
+  # this date is specific since lubridate::date() is different from base::as.Date()
+  # since as.Date returns the UTC date and date() doesn't
+  test_df <- tibble(
+    a = as.POSIXct(c("2012-03-26 23:12:13", NA), tz = "America/New_York"))
+
+  # we can't (for now) use namespacing, so we need to make sure lubridate::date()
+  # and not base::date() is being used
+  # TODO remove once https://issues.apache.org/jira/browse/ARROW-14575 is done
+  date <- lubridate::date
```

Comment:
This is only necessary in our testing — right? Something about the way that `compare_dplyr_binding()` or testthat works makes it so we're not getting `lubridate::date()` by default here even though presumably we should since we  call `library(lubridate, warn.conflicts = FALSE)` at the top of this file, yeah?

Reply:
As far as I can tell, that is correct. I never had this issue in an interactive session after loading lubridate I don't think we will be 100% sure until we support namespacing with `pkg::function()` syntax.